python/flexflow_c.cc

```diff
@@ -1864,7 +1864,7 @@ DLRMConfig::DLRMConfig(void)
       continue;
     }
     if (!strcmp(argv[i], "--arch-embedding-size")) {
-      std::stringstream ss(std::string(argv[++i]));
+      std::stringstream ss((std::string(argv[++i])));
```

Comment:
Why extra parens?

Reply:
Without the parens, the compiler interprets it as a function prototype